examples/notebooks/refractiveindex_info_db_example.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1> Refractive Index Information DB </h1>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from solcore.absorption_calculator.nk_db import download_db, search_db\n",
+    "from solcore import material\n",
+    "from solcore import si\n",
+    "from solcore.solar_cell import SolarCell\n",
+    "from solcore.structure import Layer\n",
+    "from solcore.solar_cell_solver import solar_cell_solver, default_options\n",
+    "\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wl = si(np.arange(100, 900, 10), 'nm')\n",
+    "\n",
+    "opts = default_options\n",
+    "opts.optics_method = 'TMM'\n",
+    "opts.wavelength = wl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download the database from refractiveindex.info. This only needs to be done once.\n",
+    "Can specify the source URL and number of interpolation points.\n",
+    "download_db()\n",
```

Comment:
"download_db()" should be in a code fragment, not the markdown section

Reply:
done.